pkg/config/config_test.go

```diff
@@ -340,8 +345,12 @@ integrations:
 				Metrics: metricsserver.Options{
 					BindAddress: configapi.DefaultMetricsBindAddress,
 				},
-				LeaderElectionID: "",
-				LeaderElection:   false,
+				LeaderElection:             true,
+				LeaderElectionID:           configapi.DefaultLeaderElectionID,
+				LeaderElectionResourceLock: resourcelock.LeasesResourceLock,
+				LeaseDuration:              ptr.To(15 * time.Second),
```

Comment:
Maybe instead of the magic values refer to values in `defaultControlOption`, so that is easier to identify which fields deviate from defaults?